browser/templates/browser/data_centres/ceda.html

```diff
@@ -0,0 +1,183 @@
+                    aria-expanded="false" aria-controls="readmeContent">
+                00README
+            </button>
+        </div>
+    </div>
+    <div class="row">
+        <div class="collapse col mb-2" id="readmeContent">
+            <div class="card card-body">
+
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col messages"></div>
+    </div>
+    <div class="alert alert-light text-center text-dark {% if path == "/" %} hide {% endif %}" id="page_load" role="alert">
+        Loading...
+    </div>
+    <table class="table table-sm">
+        <thead>
+        <th style="width:25.3%"><span class="text-muted small" id="dir_count">0 dirs</span>&nbsp;<span
+                class="text-muted small" id="file_count">0 files</span></th>
+        <th style="width:41.5%">Description</th>
+        <th style="width:8.3%">Size</th>
+        <th style="width:16.6%">Actions</th>
+        </thead>
+
+
+        <tbody id="results"></tbody>
+    </table>
+    <div class="row">
+        <div class="col messages"></div>
+    </div>
+
+    <script src="{% static 'browser/js/mustache.min.js' %}"></script>
+    <script src="{% static 'browser/js/browse.js' %}?version=1.2"></script>
+    <script src="{% static 'browser/js/elastic_browser.js' %}?version=2.0"></script>
+
+    <script>
+
+        $(document).ready(function () {
+            // Setup the page and load the data
+            // Set options
+            var options = {
+                path_prefix: DOWNLOAD_SERVICE,
+                max_files_per_page: "{{MAX_FILES_PER_PAGE}}"
+            };
+            // Load the data
+            try {
+                ElasticBrowser.setup(options);
+                ElasticBrowser.addResults();
+            } catch(err){
+                var page_load = $('#page_load')
+                page_load.html("An error occured. Some older browsers are unsupported, try a newer browser. If the problem continues, <a href='mailto:support@ceda.ac.uk'>contact support.</a>")
+                page_load.removeClass('alert-light text-dark')
+                page_load.addClass('alert-danger')
+                page_load.show()
+            }
+        })
+    </script>
+
+
+    {# Mustache templates #}
+    <script id="row_template" type="text/html">
+    <tr <% &row_attrs %> >
+        <td class="align-middle"><% &icon %>&nbsp;<% &item %></td>
+        <td class="align-middle"><% &description %></td>
+        <td class="align-middle"><% size %></td>
+        <td class="align-middle"><% &actions %></td>
+    </tr>
+    </script>
+
+    <script>
+        var DOWNLOAD_SERVICE = "{{ DOWNLOAD_SERVICE }}";
+        var USE_FTP = {% if USE_FTP %}true;{% else %}false;{% endif %}
+    </script>
+
+
+{% endblock %}
+{% block tail_js_extra %}
+    <script>
+        $('body').tooltip({
+            selector: '[data-toggle="tooltip"]'
+        });
+    </script>
+{% endblock %}
```

Comment:
This ceda.html template has some redundant stuff from browse_base.html, e.g. the "Bulk Download Options" block. Could you make this one simpler like the others?